docs/extending/admin_views.md

```diff
@@ -102,6 +97,17 @@ Now create a `templates/wagtailcalendar/` folder within the `wagtailcalendar` ap
 {% endblock %}
 ```
 
```

Comment:
I think this needs to be wrapped in code blocks but I can do when merging if needed.